templates/solutions/telco/5g-core.html

```diff
@@ -0,0 +1,595 @@
+                        attrs={"class": "p-logo-section__logo"}) | safe
+            }}
+            <hr class="p-rule--muted" />
+            <p>
+              <a href="https://sylvaproject.org/">Learn more&nbsp;&rsaquo;</a>
+            </p>
+          </div>
+          <div class="p-logo-section__item">
+            {{ image(url="https://assets.ubuntu.com/v1/392ce2d3-ETI-member-logo.png",
+                        alt="ETI",
+                        width="192",
+                        height="256",
+                        hi_def=True,
+                        loading="lazy",
+                        attrs={"class": "p-logo-section__logo"}) | safe
+            }}
+            <hr class="p-rule--muted" />
+            <p>
+              <a href="https://www.etsi.org/">Learn more&nbsp;&rsaquo;</a>
+            </p>
+          </div>
+          <div class="p-logo-section__item">
+            {{ image(url="https://assets.ubuntu.com/v1/45f80dee-cloud-native-logo@2x.png",
+                        alt="",
+                        width="280",
+                        height="209",
+                        hi_def=True,
+                        loading="lazy",
+                        attrs={"class": "p-logo-section__logo"}) | safe
+            }}
+            <hr class="p-rule--muted" />
+            <p>
+              <a href="https://www.cncf.io/">Learn more&nbsp;&rsaquo;</a>
+            </p>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+
+  <div class="p-section">
+    <div class="row">
+      <hr />
+      <div class="p-section--shallow">
+        <h2>One subscription for security, compliance and support across the stack</h2>
+      </div>
+    </div>
+    <div class="row--50-50">
+      <div class="col">
+        <p>
+          Ubuntu Pro offers comprehensive support and advanced features designed to meet the stringent requirements of 5G Core networks.
+        </p>
+        <p>
+          With extended security maintenance, real-time kernel support, and automation tools, Ubuntu Pro ensures that telecom infrastructures remain secure, efficient, and reliable.
+        </p>
+      </div>
+      <div class="col">
+        <div class="u-embedded-media">
+          <iframe title="Ubuntu Pro explained in 2 minutes"
+                  class="u-embedded-media__element"
+                  src="https://www.youtube.com/embed/tNECRG5r2sg?si=B98-42RGdgpJebjQ"
+                  allowfullscreen=""></iframe>
+        </div>
+      </div>
+    </div>
+  </div>
+
+  <div class="p-section">
+    <div class="row">
+      <hr />
+      <div class="col-3">
+        <div class="p-section--shallow">
+          <h2>Learn more</h2>
+        </div>
+      </div>
+      <div class="col-9">
+        <div class="p-section--shallow">
+          <div class="row">
+            <div class="col-3">
+              <h3 class="p-heading--5">Webinars</h3>
+            </div>
+            <div class="col-6">
+              <div class="row">
```

Comment:
I think we should use semantic markup where when we can and divided ul instead several rows would serve that purpose here (I also don't think using rows this way is valid vanilla, but my greater concern is making sure we use semantic html)